azure-pipelines.yml

```diff
@@ -1,6 +1,24 @@
 #=============================================================================
 # Azure Pipeline NMODL settings
 #=============================================================================
+variables:
+- name: ReleaseWheelBuild
+  value: false
+- name: NightlyWheelBuild
+  value: false
+- name: buildWheel
+  value: $[or(in(variables['Build.Reason'], 'Schedule'),
+              and(in(variables['Build.Reason'], 'Manual'), eq(variables.NightlyWheelBuild, true)))]
+
```

Comment:
I am wondering when/how `NightlyWheelBuild` variable becomes True.

Reply:
This is a user variable that you set whenever you do a manual build that should produce a wheel. The aim is that not every manual build shoudl produce. Only when you set the variable to true when you start the build you get a wheel

Reply:
IIUC, wheel won't be uploaded here unless NightlyWheelBuild is set explicitly
 In case of nightly wheel we want to upload it for every scheduled build. So we can do opposite condition here i.e. don't upload if user asked not to do. (see neuron)

Reply:
I made a mistake here ! (I didn't read `$[or` at the beginning). Makes perfect sense!